sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md

```diff
@@ -1,11 +1,10 @@
 # Release History
 
-## 4.0.0-beta.4 (Unreleased)
+## 4.0.0-beta.4 (2022-06-07)
 
 ### Features Added
 - Added `Kind` property to the `DocumentPage` class.
 - Added the `Paragraphs` property to the `AnalyzeResult` class. This property holds information about the paragraphs extracted from the input documents.
-- Added `Caption` and `Footnotes` properties to `DocumentTable`.
```

Comment:
Service confirmed this feature won't be supported. Since it's still present in the swagger, the feature crew decided to keep it in the public API for this beta, but we'll hide it from intellisense.